cluster/canary/tabulator.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testgrid-tabulator
+  namespace: testgrid-canary
+  labels:
+    app: testgrid
+    channel: stable
+    component: tabulator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: testgrid
+      channel: stable
+      component: tabulator
+  template:
+    metadata:
+      labels:
+        app: testgrid
+        channel: stable
+        component: tabulator
+    spec:
+      serviceAccountName: tabulator
+      containers:
+      - name: tabulator
+        image: gcr.io/k8s-testgrid/tabulator:latest # TODO(michelle192837): pin to a version that contains this PR
```

Comment:
🤷

Reply:
Was wondering, but figured I should be safe 😅 Maybe I'll try that for the prod deployment.

I'll add a short doc under cluster/ for "adding a new deployment" instructions too once I've got it straightened out. (I will never remember where iam binding stuff lives)

Reply:
Please use a specific version, not latest.

Reply:
Since this has a bad version, it will not autobump. This is a bad process.

The process should be to:
1) Get a version to prod that includes this image
2) Gain confidence we will back prod to a version that does not include this image
3) Add the new deployment at this version
4) Verify this new deployment is working correctly
5) Start adding production dependencies on this deployment

Reply:
👍  https://github.com/GoogleCloudPlatform/testgrid/pull/828

Reply:
So 1 and 2 should be manually deploying the component in canary (to test) and prod (to test again), until we're sure about the deployment, _then_ submit these yaml files (for canary + prod at the same time should be fine, since it'll be manually tested in both?)

Reply:
Ideally nothing is deployed manually.